crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -24,6 +24,7 @@ enum UnusedImportContext {
     Init {
         first_party: bool,
         dunder_all_count: usize,
+        ignore_init_module_imports: bool,
```

Comment:
This field is required to make fix titles follow the "old" behavior in `__init__.py` files. When we finally deprecate the option `ignore_init_module_imports` then this field `ignore_init_module_imports` can be deleted too.